src/audio/rampTable.js

```diff
@@ -0,0 +1,11 @@
+export default (start, end, length) => {
+  const output = Array(length)
+  const delta = end - start
+  const step = delta / length
+
+  for (let i = 0; i < length; i++) {
```

Comment:
can't believe you wouldn't do this in a map:

`return Array(length).fill(null).map( ( _ , index ) => (step * index) + start)`